### ISSUE TITLE
`/opt/dev-scripts` not created automatically

### ISSUE THREAD

**Issue by jhutar** [openshift-metalkube/dev-scripts#207]
On a clean CoreOS 7.6 system, following README.md instructions, `make` fails because of missing directory:

```
[tester@... dev-scripts]$ make
./01_install_requirements.sh
+ source common.sh
++++ dirname common.sh
+++ cd .
+++ pwd
++ SCRIPTDIR=/home/tester/dev-scripts
+++ whoami
++ USER=tester
++ '[' -z '' ']'
++ '[' -f /home/tester/dev-scripts/config_tester.sh ']'
++ echo 'Using CONFIG /home/tester/dev-scripts/config_tester.sh'
Using CONFIG /home/tester/dev-scripts/config_tester.sh
++ CONFIG=/home/tester/dev-scripts/config_tester.sh
++ source /home/tester/dev-scripts/config_tester.sh
+++ set +x
++ ADDN_DNS=
++ EXT_IF=
++ PRO_IF=
++ INT_IF=
++ ROOT_DISK=/dev/vda
++ FILESYSTEM=/
++ WORKING_DIR=/opt/dev-scripts
++ NODES_FILE=/opt/dev-scripts/ironic_nodes.json
++ NODES_PLATFORM=libvirt
++ MASTER_NODES_FILE=ocp/master_nodes.json
++ export RHCOS_IMAGE_URL=https://releases-rhcos.svc.ci.openshift.org/storage/releases/maipo/
++ RHCOS_IMAGE_URL=https://releases-rhcos.svc.ci.openshift.org/storage/releases/maipo/
++ export RHCOS_IMAGE_VERSION=47.284
++ RHCOS_IMAGE_VERSION=47.284
++ export RHCOS_IMAGE_NAME=redhat-coreos-maipo-47.284
++ RHCOS_IMAGE_NAME=redhat-coreos-maipo-47.284
++ export RHCOS_IMAGE_FILENAME=redhat-coreos-maipo-47.284-qemu.qcow2
++ RHCOS_IMAGE_FILENAME=redhat-coreos-maipo-47.284-qemu.qcow2
++ export RHCOS_IMAGE_FILENAME_OPENSTACK=redhat-coreos-maipo-47.284-openstack.qcow2
++ RHCOS_IMAGE_FILENAME_OPENSTACK=redhat-coreos-maipo-47.284-openstack.qcow2
++ export RHCOS_IMAGE_FILENAME_DUALDHCP=redhat-coreos-maipo-47.284-dualdhcp.qcow2
++ RHCOS_IMAGE_FILENAME_DUALDHCP=redhat-coreos-maipo-47.284-dualdhcp.qcow2
++ export RHCOS_IMAGE_FILENAME_LATEST=redhat-coreos-maipo-latest.qcow2
++ RHCOS_IMAGE_FILENAME_LATEST=redhat-coreos-maipo-latest.qcow2
++ export IRONIC_IMAGE=quay.io/metalkube/metalkube-ironic
++ IRONIC_IMAGE=quay.io/metalkube/metalkube-ironic
++ export IRONIC_INSPECTOR_IMAGE=quay.io/metalkube/metalkube-ironic-inspector
++ IRONIC_INSPECTOR_IMAGE=quay.io/metalkube/metalkube-ironic-inspector
++ export IRONIC_DATA_DIR=/opt/dev-scripts/ironic
++ IRONIC_DATA_DIR=/opt/dev-scripts/ironic
++ export KUBECONFIG=/home/tester/dev-scripts/ocp/auth/kubeconfig
++ KUBECONFIG=/home/tester/dev-scripts/ocp/auth/kubeconfig
++ export 'SSH=ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ConnectTimeout=5'
++ SSH='ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ConnectTimeout=5'
++ export LIBVIRT_DEFAULT_URI=qemu:///system
++ LIBVIRT_DEFAULT_URI=qemu:///system
++ '[' tester '!=' root -a '' == /run/user/0 ']'
++ [[ 1001 -eq 0 ]]
++ sudo -n uptime
+++ awk -F= '/^ID=/ { print $2 }' /etc/os-release
+++ tr -d '"'
++ [[ centos != \c\e\n\t\o\s ]]
+++ awk -F= '/^VERSION_ID=/ { print $2 }' /etc/os-release
+++ tr -d '"'
++ [[ 7 -ne 7 ]]
+++ df / --output=fstype
+++ grep -v Type
++ FSTYPE=xfs
++ case ${FSTYPE} in
+++ xfs_info /
+++ grep -q ftype=1
++ [[ -n '' ]]
++ '[' 2710 = 0 ']'
++ '[' '!' -d /opt/dev-scripts ']'
++ echo 'Creating Working Dir'
Creating Working Dir
++ mkdir /opt/dev-scripts
mkdir: cannot create directory ‘/opt/dev-scripts’: Permission denied
make: *** [requirements] Error 1
```

When I create it manually and rerun `make`, it proceeds further.

**Comment by jhutar** [comment]
Hmm, this is probably resolved by https://github.com/openshift-metalkube/dev-scripts/pull/205

**Comment by hardys** [comment]
We merged #199 which I think resolves this, please reopen if not, thanks!
